ERROR:root:Unable to get the submission from the db
Traceback (most recent call last):
  File "/home/willy/Development/reddit-groupme-bot/src/database_client.py", line 30, in getSubmissionById
    submissions_collection = self.db['submission']
NameError: name 'self' is not defined
ERROR:root:Unable to insert into db
Traceback (most recent call last):
  File "/home/willy/Development/reddit-groupme-bot/src/database_client.py", line 22, in insertSubmission
    id = submissions_collection.insert_one(submission)
  File "/home/willy/anaconda3/lib/python3.6/site-packages/pymongo/collection.py", line 684, in insert_one
    common.validate_is_document_type("document", document)
  File "/home/willy/anaconda3/lib/python3.6/site-packages/pymongo/common.py", line 453, in validate_is_document_type
    "collections.MutableMapping" % (option,))
TypeError: document must be an instance of dict, bson.son.SON, bson.raw_bson.RawBSONDocument, or a type that inherits from collections.MutableMapping
